<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CQDlgImageColor</class>
 <widget class="QDialog" name="CQDlgImageColor">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>233</width>
    <height>134</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Default Image Color</string>
  </property>
  <property name="locale">
   <locale language="English" country="UnitedStates"/>
  </property>
  <widget class="QCheckBox" name="qqFromFogColor">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>221</width>
     <height>18</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::StrongFocus</enum>
   </property>
   <property name="text">
    <string>Use environment fog color</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>30</y>
     <width>111</width>
     <height>18</height>
    </rect>
   </property>
   <property name="text">
    <string>Red</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="qqRed">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>30</y>
     <width>76</width>
     <height>18</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>50</y>
     <width>111</width>
     <height>18</height>
    </rect>
   </property>
   <property name="text">
    <string>Green</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="qqGreen">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>50</y>
     <width>76</width>
     <height>18</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>70</y>
     <width>111</width>
     <height>18</height>
    </rect>
   </property>
   <property name="text">
    <string>Blue</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="qqBlue">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>70</y>
     <width>76</width>
     <height>18</height>
    </rect>
   </property>
  </widget>
  <widget class="QDialogButtonBox" name="qqOkCancel">
   <property name="geometry">
    <rect>
     <x>15</x>
     <y>100</y>
     <width>201</width>
     <height>32</height>
    </rect>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
